prow/scripts/cluster-integration/helpers/provision-gke-cluster.sh

```diff
@@ -35,10 +35,14 @@ if [ "${discoverUnsetVar}" = true ] ; then
     exit 1
 fi
 
+readonly CURRENT_TIMESTAMP=$(date +%Y%m%d)
+readonly TTL_HOURS="3"
+
 CLUSTER_VERSION_PARAM="--cluster-version=1.12"
 MACHINE_TYPE_PARAM="--machine-type=n1-standard-2"
 NUM_NODES_PARAM="--num-nodes=3"
 NETWORK_PARAM=(--network=default)
+ADDITIONAL_LABELS="created-at=${CURRENT_TIMESTAMP},ttl=${TTL_HOURS}"
```

Comment:
Please use `ADDITIONAL_LABELS_PARAM` not to overwrite if it's been set outside

Reply:
Changed it to still be able to use ADDITIONAL_LABELS and introduced CLEANER_LABELS instead.